bowtie/templates/report.html.j2

```diff
@@ -363,6 +367,29 @@
           </div>
         </div>
       </div>
+
+
+      <div class="modal fade" role="dialog" id="implementation-{{ loop.index }}-runtime-info" tabindex="-1" aria-labelledby="implementation-{{ loop.index }}-runtime-info-label" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="implementation-{{ loop.index }}-runtime-info-label">
+                <b>{{ implementation.name }} </b> <small class="text-muted"> {{ implementation.language }}</small>
+              </h5>
+              <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <p><strong>OS Version : </strong>{{ implementation.get("os_version", "") }}</p>
```

Comment:
There are extra spaces in front of the `:`'s here, can you trim those.

Also maybe let's only show values where they're non-empty (so hide any row where we don't have a value for that key).

Reply:
Extra white space is removed!
Conditionally rendering the row if its value is non-empty